### ISSUE TITLE
`gdown` does not seem to be working

### ISSUE THREAD

**Issue by sarthakpati** [mlcommons/GaNDLF#765]
**Describe the bug**
Current CI seems to be broken.

**To Reproduce**
1. Run any CI test
2. See error:
```python-traceback
[SNIP!]
        if gdrive_file_id and is_gdrive_download_link:
            content_disposition = six.moves.urllib_parse.unquote(
                res.headers["Content-Disposition"]
            )
            m = re.search(r"filename\*=UTF-8''(.*)", content_disposition)
>           filename_from_url = m.groups()[0]
E           AttributeError: 'NoneType' object has no attribute 'groups'
```
Example: https://github.com/mlcommons/GaNDLF/actions/runs/7489779631/job/20387346791?pr=764#step:9:219

**Expected behavior**
The sample data file download should work.

**Screenshots**
N.A.

**GaNDLF Version**
Current master

**Desktop (please complete the following information):**
N.A.

**Additional context**
Basically, it is this error: https://github.com/wkentaro/gdown/issues/291
